packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/drop/useDrop.tsx

```diff
@@ -102,8 +89,11 @@ export const useDrop = (): UseDropValue => {
       if (data === '') {
         return;
       }
+      const dropPosition = reactflow.screenToFlowPosition({
```

Comment:
This feature is interesting, don't forget to mention it in your ReactFlow documentation